middlewares/errorHandler.test.js

```diff
@@ -47,110 +33,182 @@ describe('errorHandler', () => {
             },
             displayResponseErrors: true
         });
-
-        response._headers = {}; // eslint-disable-line
-        response.headersSent = undefined;
-        response.data = undefined;
-        response.set.mockReset();
-        response.set.mockImplementation((name, value) => {
-            response._headers[name] = value; // eslint-disable-line
-        });
-        response.send.mockReset();
-        response.status.mockReset();
-        response.status.mockImplementation(() => response);
-        response.statusCode = undefined;
-
         next.mockReset();
-
-        report.mockReset();
+        end.mockReset();
+        setHeader.mockReset();
+
+        req = httpMocks.createRequest();
+        // to deactivate logs stdout
+        req.log = {
+            trace: jest.fn(),
+            debug: jest.fn(),
+            warn: jest.fn(),
+            error: jest.fn(),
+            info: jest.fn()
+        };
+
+        res = httpMocks.createResponse();
+        res.end = end;
```

Comment:
```suggestion
        res.end = jest.fn();
```
comme ca tu vires le `let end = jest.fn()` et `end.mockReset()`